### PR TITLE
Added Indian University NIRF Ranking in Python Scripts

### LINKED ISSUE #783: Indian University NIRF Ranking
In this script,
will create an application that will allow user to get the NIRF Ranking of any top university in India.

This will include almost 200 top universities of India.

Along with NIRF Ranking, this application will also tells us about 5 things,
1) Teaching, Learning & Resources (TLR)
2) Research and Professional Practice (RPC)
3) Graduation Outcomes (GO)
4) Outreach and Inclusivity (OI)
5) Peer Perception

The NIRF Ranking of any university is decided on the basis of these above 5 values.

### PR DESCRIPTION
**Pull request by akash435**
- A "Indian University NIRF Ranking" is an application created in python with tkinter gui.
- In this application, user can get the NIRF Ranking of TOP 200 University.
- Along with NIRF, other details like, Institute ID, Teaching, Learning and Resources (TLR), Research and Professional Practice (RPC), Graduation Outcomes (GO), Outreach and Inclusivity, Peer Perception, City and State.

Fixes #783

## Checklist

- [x] The PR title starts with "Fix #issue_number".  
- [x] My branch is up-to-date with the main branch.
- [x] The PR does not contain any unnecessary auto-generated code.
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the guidelines of the project.
- [x] The PR is raised after the issue has been assigned to the PR creator.

Below are some screenshots showing how application works :

### CODE REVIEW

**Comment by akash435** [comment]
@GowthamGoush @RaveenaBhasin , creates the pr for adding "Indian University NIRF Ranking" under Python Scripts, please check it.

File: Python/Indian University NIRF Ranking/indian_university_NIRF_ranking.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+rpc = data['rpc'].tolist()
+go = data['go'].tolist()
+oi = data['oi'].tolist()
+perception = data['perception'].tolist()
+city = data['city'].tolist()
+state = data['state'].tolist()
+rank = data['rank'].tolist()
+
+
+# print(len(name))
```
Comment (GowthamGoush) [inline Python/Indian University NIRF Ranking/indian_university_NIRF_ranking.py:49]: You may remove these unnecessary comments.

**Review by GowthamGoush — CHANGES_REQUESTED** [review]
Please consider removing unnecessary comments.

**Comment by akash435** [comment]
@GowthamGoush , done the requested changes.

**Review by GowthamGoush — APPROVED** [review]
Ready to be merged.

**Pull request merged**